src/main/java/skye/commands/AddTaskCommand.java

```diff
@@ -0,0 +1,35 @@
+package skye.commands;
+
+import java.io.IOException;
+
+import skye.data.TaskList;
+import skye.data.task.Task;
+import skye.storage.Storage;
+import skye.ui.UI;
+
+/**
+ * Represents a generic command to handle adding of Task objects and its subclasses.
+ */
+public class AddTaskCommand extends Command {
```

Comment:
Really like how you broke down the commands into the different specific command groups:)